### PR TITLE
refactor: minor naming changes and updates in specs

### PR DESCRIPTION
**Pull request by ninabarbakadze**
## Overview

follow-up pr addressing some of the comments on [minfee-module](https://github.com/celestiaorg/celestia-app/pull/3094)

### CODE REVIEW

**Review by ninabarbakadze — COMMENTED** [review]
markdown-linter throws error because `minfee/readme.md` link doesn't exist yet but it will after this pr is merged.

File: specs/src/specs/state_machine_modules.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Celestia app is built using the cosmos-sdk, and follows standard cosmos-sdk modu
 - [paramfilter](https://github.com/celestiaorg/celestia-app/blob/main/x/paramfilter/README.md)
 - [upgrade](https://github.com/celestiaorg/celestia-app/blob/main/x/upgrade/README.md)
 - [tokenfilter](https://github.com/celestiaorg/celestia-app/blob/main/x/tokenfilter/README.md)
+- [minfee](https://github.com/celestiaorg/celestia-app/blob/main/x/minfee/README.md)
```
Comment (rootulp) [inline specs/src/specs/state_machine_modules.md:13]: 🙏 please alphabetize

File: x/minfee/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# `x/minfee`
+
+## Abstract
+
+The `x/minfee` module is responsible for managing the gov-modifiable constant `GlobalMinGasPrice` introduced in v2 within the network.`GlobalMinGasPrice` ensures that all transactions adhere to this global minimum threshold, which is set in the genesis file and can be updated via governance proposals.
```
Comment (rootulp) [inline x/minfee/README.md:5]:
```suggestion
The `x/minfee` module is responsible for managing the gov-modifiable parameter `GlobalMinGasPrice` introduced in app version 2. `GlobalMinGasPrice` ensures that all transactions adhere to this global minimum threshold, which is set in the genesis file and can be updated via governance proposals.
```

File: x/minfee/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
```
Comment (rootulp) [inline x/minfee/README.md:6]:
```suggestion
```

**Review by rootulp — APPROVED** [review]

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**